### DISCUSSION TITLE
trivial edit needed on readme.md

### DISCUSSION THREAD

**Discussion by degrees57** [code4recovery/12-step-meeting-list#472] [General]
The URL in the readme file displayed here in github has a missing letter, so the redirect goes 404.

Is `https://github.com/code4recovery/12-step-meeting-list/dicussions`
Should be `https://github.com/code4recovery/12-step-meeting-list/discussions`

The first letter "s" is missing in "discussions" at the end of the URL

I don't know much about github to know if I can just edit the file (like with wikipedia), but I doubt it.  So I guess this is the next best way to get the update made?  Thank you for the whole TSML plugin, it is wonderful.

**Comment by tech2serve** [comment]
Thanks for the catch! That was my mistake. We should be able to get it fixed in the next release.
